scoreboard/utils.py

```diff
@@ -118,6 +119,18 @@ def absolute_url(path):
     """Build an absolute URL.  Not safe for untrusted input."""
     return urlparse.urljoin(flask.request.host_url, path)
 
+def normalize_input(answer):
+    """"Take a string and normalize it to a standard format."""
+
+    """Strip leading and trailing whitespace characters"""
+    match = re.match("^\\s*(.*\\S)\\s*$", answer)
```

Comment:
For just removing leading and trailing whitespace, why not use .strip()?

I think this can be as simple as:

    return answer.strip().lower()